R/data.R

```diff
@@ -0,0 +1,32 @@
+#' Superheroes Data
+#'
+#' A dataset of superheroes in a REDCapTidieR `supertbl` object
+#'
+#' @format
+#' ## `heroes_information`
+#' A `tibble` with 734 rows and 12 columns:
+#' \describe{
+#'   \item{record_id}{REDCap record ID}
+#'   \item{name}{Hero name}
+#'   \item{gender}{Gender}
+#'   \item{eye_color}{Eye color}
+#'   \item{race}{Race}
+#'   \item{hair_color}{Hair color}
+#'   \item{height}{Height}
+#'   \item{weight}{Weight}
+#'   \item{publisher}{Publisher}
+#'   \item{skin_color}{Skin color}
+#'   \item{alignment}{Alignment}
+#'   \item{form_status_complete}{REDCap instrument completed?}
+#' }
+#'
+#' ## `super_hero_powers`
+#' A `tibble` with 5,966 rows and 4 columns:
+#' \describe{
+#'   \item{record_id}{REDCap record ID}
+#'   \item{redcap_repeat_instance}{REDCap repeat instance}
+#'   \item{power}{Super power}
+#'   \item{form_status_complete}{REDCap instrument completed?}
+#' }
+#' @source <https://www.superherodb.com/>
+"superheroes_supertbl"
```

Comment:
Is this something we want to have exported to users? I believe as it is right now this will be available as `REDCapTidieR::superheroes_supertbl`, but is it meant to be internal?

Reply:
Maybe I'm wrong but I think this needs to be external external so we can use it in examples. I guess alternatively we could reference it in the examples like:

```r
superheroes_supertbl <- readRDS(system.file("testdata/superheroes_supertbl.RDS", package = "REDCapTidieR"))
```

but it won't look the best in our docs. Are there other options I'm missing?

Reply:
Ah, I didn't realize this would be a part of "option 3", my fault entirely. I think it's fine, we rely on it so heavily for our example documentation and you have the help documentation available to support it.